h/streamer/metrics.py

```diff
@@ -33,14 +33,15 @@ def websocket_metrics(queue):
 def metrics_process(registry, queue):  # pragma: no cover
     session = db.get_session(registry.settings)
 
-    newrelic.agent.initialize()
+    newrelic.agent.initialize(
+        config_file=resource_filename("h.streamer", "conf/newrelic.ini"),
+        log_file="debug",
+    )
+    newrelic.agent.register_application(timeout=5)
```

Comment:
We weren't registering the application before. Apparently we should be to have it read the config and enable itself. Otherwise it just silently doesn't do anything.